tests/lib/kubernetes/caasp.py

```diff
@@ -50,14 +51,16 @@ def __init__(self, workspace: Workspace, hardware: HardwareBase):
     def bootstrap(self):
         super().bootstrap()
         self.hardware.ansible_run_playbook('playbook_caasp.yaml')
-        self.workspace.execute("skuba cluster init --control-plane "
+        # Ansible retrieves the binary locally
+        os.chmod(self._skuba, 0o755)
```

Comment:
Can you set that in the ansible download code so it's correctly done from the beginning?

Reply:
The fetch function does not support that. There is a feature request about it on ansible.

Reply:
but the file_module does. so fixing after fetch makes sense imo

Reply:
Ok I did not like the delegate option but for no good reason.
Updated to use ansible.